src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -1354,6 +1355,88 @@ To pass child chart options through the command line, you need to prefix them wi
 
 </TabsPageItem>
 
+<TabsPageItem id="5">
+
+There are three ways to install the Kubernetes integration in your GKE Autopilot cluster.     
+
+<CollapserGroup>
+    <Collapser
+        id="guided-gke"
+        title="Guided install (recommended)"
+    >
+
+Follow the instructions in the **Guided install** tab above. 
+
+The guided install will ask if you are installing in a GKE Autopilot cluster. After you enter `Y`, we will set `privileged`, `controlPlane`, `logging`, and `pixie` to `false`. The install will also set the kubelet scheme to http and kubelet port to 10255. 
+
+
+    </Collapser>
+    <Collapser
+        id="helm-gke"
+        title="Manual Helm"
```

Comment:
If the other two collapsers are duplicative of content in the other tabs, we might consider prioritizing these steps first ahead of guided install / helm

Reply:
OK It took a min to grok but I see the differentiation. I'd still add some context about the action users are going to do that are ~different from the horiz tabs, just to avoid confusion. Suggestion tho! I'm not the intended audience